packages/plugins/rubygem-smart_proxy_dns_infoblox/rubygem-smart_proxy_dns_infoblox.spec

```diff
@@ -18,7 +18,7 @@
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
 Version: 1.0.0
-Release: 3%{?foremandist}%{?dist}
+Release: 3.1%{?foremandist}%{?dist}
```

Comment:
small bump is intentional, as 4 exists already in koji

Reply:
The foremandist should be defined differently and thus end up with a different package that rpm/develop right?

Reply:
Yeah, the foremandist would save you, but as a user, I'd expect `package-1.2.3-4.el7` and `package-1.2.3-4.el8` being the same source, built for different dists. As a -4 already exists, we would pretend that this -4 and the existing -4 are the same, but built for 2.0 vs 2.1, which they won't be.

Reply:
Can't we pick the -4 change and go -5?

Reply:
We could.

Reply:
Well, we could, but then we'd also have to pick some newer changes as that in -4 is broken.